en/key_concepts.rst

```diff
@@ -9,9 +9,10 @@ Nodes
 Nodes are mainly pages that the visitors can see on the website, they are containers for zones and blocks.
 There are two types of nodes in Open Orchestra:
 * Nodes that represent pages of the website
-* The transverse node (which is unique) containing the transverse blocks (blocks that are common to multiple pages of a website, for instance the menu block).
+* The transverse node (there is no version but one per language) containing the transverse blocks
```

Comment:
Sounds weird. I would remove the parenthesis here, and add a second sentence at the end
Something like "this node is not versionable, and there can only be once transverse node in each language of the site".
What do you think @nicolasThal @itkg-bfouche ?